File: src/main/java/com/sudosoo/takeiteasy/controller/MessageController.java
package com.sudosoo.takeiteasy.controller;

import com.sudosoo.takeiteasy.dto.message.MessageSendRequestDto;
import com.sudosoo.takeiteasy.service.MessageService;
import jakarta.validation.Valid;
import lombok.RequiredArgsConstructor;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/message")
@RequiredArgsConstructor
public class MessageController {
    private final MessageService messageService;
    @PostMapping(value = "/send" , name = "messageSend")
    public ResponseEntity<Void> messageSend(@RequestBody MessageSendRequestDto requestDto) {

        messageService.messageSend(requestDto);

        return ResponseEntity.ok().build();
    }


}

File: takeItEasy-presentation/src/main/kotlin/com/sudosoo/takeItEasy/controller/MessageController.kt
package com.sudosoo.takeItEasy.controller

import com.sudosoo.takeItEasy.dto.message.MessageSendRequestDto
import com.sudosoo.takeItEasy.service.MessageService
import com.sudosoo.takeiteasy.dto.message.MessageSendRequestDto
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/message")
class MessageController(val messageService: MessageService) {
    @PostMapping("/send", name = "messageSend")
    fun messageSend(@RequestBody requestDto: MessageSendRequestDto): ResponseEntity<Void> {
        messageService.send(requestDto)

        return ResponseEntity.ok().build<Void>()
    }
}